lightly_studio/src/lightly_studio/resolvers/sample_embedding_resolver.py

```diff
@@ -81,6 +82,36 @@ def get_all_by_dataset_id(
         .where(SampleEmbeddingTable.sample_id == SampleTable.sample_id)
         .where(SampleTable.dataset_id == dataset_id)
         .where(SampleEmbeddingTable.embedding_model_id == embedding_model_id)
-        .order_by(col(SampleTable.file_path_abs).asc())
+        .order_by(col(SampleTable.created_at).asc())
     )
     return list(session.exec(query).all())
+
+
+def get_hash_by_sample_ids(
+    session: Session,
+    sample_ids_ordered: list[UUID],
+    embedding_model_id: UUID,
+) -> str:
+    """Return a combined 64-bit hash for the embeddings belonging to the given samples.
+
+    The combination is deterministic with respect to the provided sample IDs.
+    """
+    if not sample_ids_ordered:
+        return "empty"
+
+    rows = session.exec(
+        select(
+            SampleEmbeddingTable.sample_id,
+            func.hash(SampleEmbeddingTable.embedding).label("h64"),
+        )
+        .where(col(SampleEmbeddingTable.sample_id).in_(set(sample_ids_ordered)))
+        .where(SampleEmbeddingTable.embedding_model_id == embedding_model_id)
+    ).all()
+
+    # Mypy does not get that 'h64' is an attribute of the returned rows
+    sample_id_to_hash = {row.sample_id: row.h64 for row in rows}  # type: ignore[attr-defined]
+    hashes_ordered = [sample_id_to_hash[sample_id] for sample_id in sample_ids_ordered]
```

Comment:
Optional: We could simplify this logic. The function argument could be `sample_ids: list[UUID]` and we could order in the db by sample_id. Then no reordering is needed.